contrib/tracking.go

```diff
@@ -0,0 +1,256 @@
+package contrib
+
+/*
+#include "tracking.h"
+*/
+import "C"
+import (
+	"image"
+
+	"gocv.io/x/gocv"
+)
+
+// Tracker - the base interface for object tracking
+//
+// s.a: https://docs.opencv.org/master/d0/d0a/classcv_1_1Tracker.html
```

Comment:
what does "s.a" stand for actually?

Reply:
see also

Reply:
ah no it makes sense, thank you! Google did not help me..

Reply:
a single   "see:"  would be better, i guess

Reply:
We often use "See also:" too